haskell.nix/library/README.md

```diff
@@ -0,0 +1,58 @@
+Template for building and testing haskell library with multiple versions of GHC via haskell.nix for Buildkite, Gitlab or GitHub CI
+
+## How to use this template
+
+- The recommended way to use this template (if you have nix flakes enabled) is `nix flake init -t github:serokell/templates#haskell-library`
+
+- Otherwise:
```

Comment:
This and the previous points seem to be duplicated with `application/README.md`. I'm not going to say that we definitely must avoid it, because I can't suggest an ideal way to do it, but if you can restructure it somehow to avoid copy-paste, it will be great.
Also the third sub-point (about `flakes` being enabled) is different, and I am not sure it's intentional (probably not).